test/core/models/ocelot/LabelSpec.scala

```diff
@@ -339,9 +339,22 @@ class LabelSpec extends BaseSpec with ProcessJson {
         "Colours"->ListLabel("Colours", List("Yellow", "Violet"))
       )
 
-      val labels = LabelCache(labelsMap, cacheMap)
+      val stack = List(
+        Flow("1", Some(LabelValue("loop", Some("One")))),
+        Flow("2", Some(LabelValue("loop", Some("One")))),
```

Comment:
I'd use Some("Two") to differentiate the first and second flows.

Reply:
updated